ci/staging-acceptance-tests.sh

```diff
@@ -0,0 +1,213 @@
+else
+  echo "❌ Org quota mismatch. Expected '$EXPECTED_ORG_QUOTA', got '$org_quota'"
+fi
+
+echo "🔍 Checking spaces for org '$ORG_NAME'..."
+spaces_output=$(cf spaces)
+space_count=$(echo "$spaces_output" | grep -c "^$SPACE_NAME$")
+if [[ "$space_count" -eq 1 ]]; then
+  echo "✅ Space '$SPACE_NAME' found."
+  cf target -s "$SPACE_NAME" >/dev/null 2>&1
+else
+  echo "❌ Expected one space named '$SPACE_NAME', found $space_count"
+fi
+
+echo "🔍 Checking space quota for space '$SPACE_NAME'..."
+space_quota=$(cf space "$SPACE_NAME" | awk -F': ' '/quota:/ {print $2}' | xargs || echo "UNKNOWN")
+if [[ "$space_quota" == "$EXPECTED_SPACE_QUOTA" ]]; then
+  echo "✅ Space quota is '$space_quota'"
+else
+  echo "❌ Space quota mismatch. Expected '$EXPECTED_SPACE_QUOTA', got '$space_quota'"
+fi
+
+echo "🔍 Verifying running security groups from 'cf space $SPACE_NAME' output..."
+space_output=$(cf space "$SPACE_NAME")
+
+for group in "${REQUIRED_SECURITY_GROUPS[@]}"; do
+  if grep -A 10 "running security groups" <<< "$space_output" | grep -q "$group"; then
+    echo "✅ Running security group '$group' is listed in space config"
+  else
+    echo "❌ Missing required running security group '$group' in space config"
+  fi
+done
+
+echo "🔍 Checking org quota resource limits from 'cf org-quota $ORG_NAME'..."
+org_quota_output=$(cf org-quota "$ORG_NAME")
+
+# Total memory
+total_memory=$(awk -F': ' '/total memory:/ {print $2}' <<< "$org_quota_output" | xargs)
+if [[ "$total_memory" == "1G" ]]; then
+  echo "✅ Total memory is 1G"
+else
+  echo "❌ Total memory mismatch: Expected '1G', got '$total_memory'"
+fi
+
+# Routes
+routes=$(awk -F': ' '/routes:/ {print $2}' <<< "$org_quota_output" | xargs)
+if [[ "$routes" == "10" ]]; then
+  echo "✅ Routes is 10"
+else
+  echo "❌ Routes mismatch: Expected '10', got '$routes'"
+fi
+
+# Service instances
+services=$(awk -F': ' '/service instances:/ {print $2}' <<< "$org_quota_output" | xargs)
+if [[ "$services" == "10" ]]; then
+  echo "✅ Service instances is 10"
+else
+  echo "❌ Service instances mismatch: Expected '10', got '$services'"
+fi
+
+PASSWORD=$(cat /dev/urandom | base64 | tr -dc '0-9a-zA-Z' | head -c32)
+echo "Creating a second CF user test.user2@fedramp.gov with a random password..."
+cf create-user "test.user2@fedramp.gov" "$PASSWORD" >/dev/null 2>&1
+
+# Sleep for 45 seconds, app runs every 30 seconds
+echo "Sleeping test for 45 seconds to wait for sandbox-bot app to process the new user..."
+sleep 45
+
+# Verify the second space was created
+
+SPACE_NAMES=("test.user" "test.user2")
+
+echo "🔍 Verifying exactly two spaces named 'test.user' and 'test.user2'..."
+spaces_output=$(cf spaces | awk 'NR>3' | xargs -n1)
+actual_spaces=($(echo "$spaces_output"))
+expected_spaces_sorted=($(printf "%s\n" "${SPACE_NAMES[@]}" | sort))
+actual_spaces_sorted=($(printf "%s\n" "${actual_spaces[@]}" | sort))
+
+if [[ "${#actual_spaces_sorted[@]}" -eq 2 && "${actual_spaces_sorted[*]}" == "${expected_spaces_sorted[*]}" ]]; then
```

Comment:
I don't necessarily have a problem with it, but if we already had tests above to verify that the `test.user` space was created, why do we need to test for it again?

Reply:
I've removed the "space" tests from the first section with the single user, now relying on the second section where the second user was added to loop and test both spaces.